data/ka'het outfits.txt

```diff
@@ -0,0 +1,194 @@
+# Copyright (c) 2019 by Tommaso Becca
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# Weapons:
+
+outfit "Ka'het Ravager Beam"
+	category "Guns"
+	cost 322000
+	thumbnail "outfit/ka'het ravager beam"
+	"mass" 13
+	"outfit space" -18
+	"weapon capacity" -18
+	"gun ports" -1
+	weapon
+		sprite "projectile/ravager beam"
+			"frame rate" 15
+		sound "disruptor"
+		"hit effect" "ravager impact"
+		"inaccuracy" .2
+		"velocity" 480
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 3.7
+		"firing heat" 1.5
+		"shield damage" 1.8
+		"hull damage" 4.3
+	description "This laser beam, once built by the Builders for use in asteroid mining, incidentally made the Ka'het very effective against a ship's armor."
+
+outfit "Ka'het Ravager Turret"
+	category "Turrets"
+	cost 417000
+	thumbnail "outfit/ka'het ravager turret"
+	"mass" 16
+	"outfit space" -22
+	"weapon capacity" -22
+	"turret mounts" -1
+	weapon
+		sprite "projectile/ravager beam"
+			"frame rate" 15
+		"hardpoint sprite" "hardpoint/ravager turret"
+		"hardpoint offset" 3.
+		sound "disruptor"
+		"hit effect" "ravager impact"
+		"inaccuracy" .2
+		"turret turn" 1.8
+		"velocity" 480
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 3.7
+		"firing heat" 1.5
+		"shield damage" 1.8
+		"hull damage" 4.3
+	description "This turret-mounted version of the Ravager Beam is one of the strongest weapons the Ka'het possess."
+
+effect "ravager impact"
+	sprite "effect/ravager impact"
+		"no repeat"
+		"frame rate" 1
+	"lifetime" 1
+	"velocity scale" 0
+
+outfit "Ka'het Annihilator"
+	category "Guns"
+	cost 351000
+	thumbnail "outfit/ka'het annihilator"
+	"mass" 10
+	"outfit space" -14
+	"weapon capacity" -14
+	"gun ports" -1
+	weapon
+		sprite "projectile/annihilator"
+			"frame rate" 7
+		sound "plasma"
+		"hit effect" "bullet impact"
+		"inaccuracy" 2.5
+		"velocity" 18
+		"lifetime" 35
+		"reload" 10
+		"firing energy" 31
+		"firing heat" 8
+		"shield damage" 43
+		"hull damage" 12
+	description "The Builders initially created the Annihilator long ago, as a countermeasure to their newly discovered shield matrix. They kept developing it for centuries, with this being its final version."
```

Comment:
It doesn't necessarily mean they just *invented* the shield matrix, but just that they found out they exist: either after a contact with the Quarg, or after the Hai attack

Reply:
You create weapons that can go against your defensive technology because at some point, you will be faced with a situation where that defensive technology fails, or in the case where someone steals your technology.
It is also the main way to refine and gauge the capabilities of your defensive measures.  Just because your new shield works at this time doesn't mean you should stop working on it until something someone else made breaks it.  At that time, you've just lost your main defense, and are probably going to be on the defensive, without a defense.

Reply:
>It doesn't necessarily mean they just invented the shield matrix, but just that they found out they exist

It does say though "their newly discovered shield matrix," implying that the shield matrix is theirs.